x-pack/plugins/security_solution/public/timelines/components/timeline/expandable_event/index.tsx

```diff
@@ -4,46 +4,83 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { find } from 'lodash/fp';
-import { EuiTextColor, EuiLoadingContent, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
 import React, { useMemo, useState } from 'react';
 
+import {
+  EuiButtonIcon,
+  EuiTextColor,
+  EuiLoadingContent,
+  EuiTitle,
+  EuiSpacer,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { find } from 'lodash/fp';
+
 import { TimelineExpandedEvent } from '../../../../../common/types/timeline';
-import { BrowserFields, DocValueFields } from '../../../../common/containers/source';
+import { BrowserFields } from '../../../../common/containers/source';
 import {
   EventDetails,
   EventsViewType,
   View,
 } from '../../../../common/components/event_details/event_details';
 import { TimelineEventsDetailsItem } from '../../../../../common/search_strategy/timeline';
-import { useTimelineEventsDetails } from '../../../containers/details';
 import * as i18n from './translations';
+import { LineClamp } from '../../../../common/components/line_clamp';
 
+export type OnEventDetailsClose = () => void;
 interface Props {
   browserFields: BrowserFields;
-  docValueFields: DocValueFields[];
+  detailsData: TimelineEventsDetailsItem[] | null;
   event: TimelineExpandedEvent;
+  isAlert: boolean;
+  loading: boolean;
   timelineId: string;
 }
 
-export const ExpandableEventTitle = React.memo(() => (
-  <EuiTitle size="s">
-    <h4>{i18n.EVENT_DETAILS}</h4>
-  </EuiTitle>
-));
+export const ExpandableEventTitle = React.memo(
+  ({
+    isAlert,
+    loading,
+    timelineId,
+    onEventDetailsClose,
+  }: {
+    isAlert: boolean;
+    loading: boolean;
+    timelineId: string;
+    onEventDetailsClose?: OnEventDetailsClose;
+  }) => {
+    return (
+      <div>
```

Comment:
is the `div` necessary here?

Reply:
yeah, if I remove it the `EuiFlexGroup` occupies too much hight of the panel.

Reply:
can we instead do like ```styled(EuiFlexGroup)` flex: 0; ````?